packages/realm/src/ObjectListeners.ts

```diff
@@ -44,7 +44,7 @@ export class ObjectListeners<T> {
           });
         } catch (err) {
           // Scheduling a throw on the event loop,
-          // since throwing synchroniously here would result in an abort in the calling C++
+          // since throwing synchronously here would result in an abort in the calling C++
```

Comment:
Thanks. Strange I didn't catch these in https://github.com/realm/realm-js/commit/789a688d69cf236b9c76457fe09c48e5feeee9c3 🤔